html/browser/delay_interactivity.js

```diff
@@ -0,0 +1,24 @@
+// Disables all links for one second after the browser window opens.
+
+(function(){
+  // If there's already an onload, let's not clobber it
+  var oldonload = window.onload
+  window.onload = function(){
+    if(typeof oldonload == 'function')
+      oldonload();
```

Comment:
... it's a single-statement if. You don't need a block for it. And we don't have any coding standards for Javascript!